torch/nn/attention/flex_attention.py

```diff
@@ -167,25 +213,25 @@ class BlockMask:
     1. (kv_num_blocks, kv_indices): Used for the forwards pass of attention, as
     we reduce along the KV dimension.
 
-    2. (q_num_blocks, q_indices): Required for the backwards pass, as computing
-    dKV requires iterating along the mask along the Q dimension.
-
-    3. [OPTIONAL] (full_kv_num_blocks, full_kv_indices): This is optional and
+    2. [OPTIONAL] (full_kv_num_blocks, full_kv_indices): This is optional and
```

Comment:
Super nit not related to this PR, the "full" naming is somewhat confusing. When I think of other sparse formats the blocks are typically the non zero values. So in my mind when I think of the blocks I think full blocks are the blocks where every value is needed for computation, not that they are "full of zeros"